runner/src/internal.rs

```diff
@@ -47,16 +47,16 @@ pub enum Command {
     BuildFunctionsExample(RunFunctionsExamples),
     BuildServer(BuildServer),
     BuildFunctionsServer(BuildFunctionsServer),
-    Format(Commits),
-    CheckFormat(Commits),
+    Format(ScopeOpt),
+    CheckFormat(ScopeOpt),
     RunTests,
-    RunCargoClippy(Commits),
+    RunCargoClippy(ScopeOpt),
     RunCargoTests(RunTestsOpt),
     RunBazelTests,
-    RunTestsTsan,
+    RunTestsTsan(ScopeOpt),
```

Comment:
Alternatively, you could just move the scope arg to top-level (even if it's not actually used by all the commands), and then the ci.yaml file becomes trivial (either add the scope or not to any command, directly in the step)

Reply:
Very good idea. Thanks. Done!